src/libraries/Microsoft.PowerFx.Core/Types/Enums/ColorTable.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.PowerFx.Core.Types.Enums
 {
     internal static class ColorTable
     {
-        public static Dictionary<string, uint> InvariantNameToHexMap { get; } = new Dictionary<string, uint>
+        public static IReadOnlyDictionary<string, uint> InvariantNameToHexMap { get; } = new Dictionary<string, uint>
```

Comment:
>IReadOnlyDictionary [](http://example.com/codeflow?start=22&length=19)

We have a *a lot* of static collection classes. We mark them as safe by using a IReadOnly* interface. 

This isn't 100% bullet proof - somebody could still cast to the mutable object and mutate.